### PR TITLE
Fix python parser on Android

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -203,9 +203,14 @@ the `freeorion-project/build` directory.
 
 Install Android NDK.
 
-Build [Python-For-Android].
+Build [Python-For-Android]. Pack standart library with
```
Comment (geoffthemedio) [inline BUILD.md]:
```suggestion
Build [Python-For-Android]. Pack standard library with
```

File: util/Directories.cpp [hunk 2]
```diff
@@ -103,6 +104,27 @@ void RedirectOutputLogAndroid(int priority, const char* tag, int fd)
     });
     background.detach();
 }
+
+void CopyInitialResourceAndroid(const std::string& rel_path)
```
Comment (geoffthemedio) [inline util/Directories.cpp:108]:
```suggestion
void CopyInitialResourceAndroid(std::string_view rel_path)
```

File: util/Directories.cpp [hunk 3]
```diff
@@ -476,6 +498,10 @@ void InitDirs(std::string const& argv0)
 
     RedirectOutputLogAndroid(ANDROID_LOG_ERROR, "stderr", 2);
     RedirectOutputLogAndroid(ANDROID_LOG_INFO, "stdout", 1);
+
+    s_python_home = s_cache_dir / "python";
+    fs::create_directories(s_python_home / "lib");
+    CopyInitialResourceAndroid("lib/python36.zip");
```
Comment (geoffthemedio) [inline util/Directories.cpp:504]: called here with `char*` literal so passing as `const string&` does redundant `string` construction probly

File: util/Directories.cpp (hunk 2 above)
Comment (o01eg) [reply util/Directories.cpp:108]: Then I got `error: invalid operands to binary expression ('fs::path' and 'std::string_view' (aka 'basic_string_view<char>'))`

File: BUILD.md (hunk 1 above)
Comment (o01eg) [reply BUILD.md]: Fixed.

File: util/Directories.cpp (hunk 2 above)
Comment (o01eg) [reply util/Directories.cpp:108]: Maybe start to use `std::filesystem::path` instead of `boost::filesystem::path` to get support for string views?

Comment (geoffthemedio) [reply util/Directories.cpp:108]: There might be compiler support issues still, eg. on older Apple Clang versions.

Edit: `std::filesystem` requires Apple Clang 11 apparently: https://en.cppreference.com/w/cpp/compiler_support

Comment (o01eg) [reply util/Directories.cpp:108]: I see, Android toolchain have only Clang 9.

Comment (o01eg) [reply util/Directories.cpp:108]: @geoffthemedio Maybe merge it as is while boost doesn't supports string view for paths?

Comment (geoffthemedio) [reply util/Directories.cpp:108]: > Android toolchain have only Clang 9.

Apple Clang and other Clang are separarte. That same table lists Clang as supporting `std::filesystem` from version 7.

> merge it as is

feel free...

**Pull request merged**